### PR TITLE
chore(build): update build tools version

### PR DESCRIPTION
**Pull request by rjuliano**
*Issue #, if available:*

*Description of changes:*
Attempt to fix the LGTM test

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ task clean(type: Delete) {
 }
 
 ext {
-    buildToolsVersion = "29.0.3"
+    buildToolsVersion = "30.0.2"
```
Comment (jamesonwilliams) [inline build.gradle:49]: Woo-hoo!

**Review by jamesonwilliams — APPROVED** [review]

**Comment by jamesonwilliams** [comment]
```console
[2020-09-25 21:35:42] [autobuild] License for package Android SDK Build-Tools 30.0.2 accepted.
[2020-09-25 21:35:42] [autobuild] Preparing "Install Android SDK Build-Tools 30.0.2 (revision: 30.0.2)".
[2020-09-25 21:35:42] [autobuild] Warning: Failed to read or create install properties file.
[2020-09-25 21:35:42] [autobuild] Checking the license for package Android SDK Platform 30 in /opt/extra/android/versions/latest/licenses
[2020-09-25 21:35:42] [autobuild] License for package Android SDK Platform 30 accepted.
[2020-09-25 21:35:42] [autobuild] Preparing "Install Android SDK Platform 30 (revision: 3)".
[2020-09-25 21:35:42] [autobuild] Warning: Failed to read or create install properties file.
[2020-09-25 21:35:42] [autobuild] FAILURE: Build failed with an exception.
[2020-09-25 21:35:42] [autobuild] * Where:
[2020-09-25 21:35:42] [autobuild] Script '/opt/src/configuration/publishing.gradle' line: 154
[2020-09-25 21:35:42] [autobuild] * What went wrong:
[2020-09-25 21:35:42] [autobuild] A problem occurred configuring project ':aws-analytics-pinpoint'.
[2020-09-25 21:35:42] [autobuild] > Failed to install the following SDK components:
[2020-09-25 21:35:42] [autobuild]       build-tools;30.0.2 Android SDK Build-Tools 30.0.2
[2020-09-25 21:35:42] [autobuild]       platforms;android-30 Android SDK Platform 30
[2020-09-25 21:35:42] [autobuild]   The SDK directory is not writable (/opt/extra/android/versions/latest)
```

File: build.gradle (hunk 1 above)
Comment (rjuliano) [reply build.gradle:49]: Dang it...it's still failing in lgtm

```
[2020-09-25 21:35:42] [autobuild] * What went wrong:
[2020-09-25 21:35:42] [autobuild] A problem occurred configuring project ':aws-analytics-pinpoint'.
[2020-09-25 21:35:42] [autobuild] > Failed to install the following SDK components:
[2020-09-25 21:35:42] [autobuild]       build-tools;30.0.2 Android SDK Build-Tools 30.0.2
[2020-09-25 21:35:42] [autobuild]       platforms;android-30 Android SDK Platform 30
[2020-09-25 21:35:42] [autobuild]   The SDK directory is not writable (/opt/extra/android/versions/latest)
[2020-09-25 21:35:42] [autobuild] * Try:
````

Comment (jamesonwilliams) [reply build.gradle:49]: [Manoel is also encountering the `The SDK directory is not writable` error](https://github.community/t/android-projects-are-not-working/131143).

**Comment by rjuliano** [comment]
The original intent was to see if this change fixed the lgtm error. However, it looks like there's [something else going on](https://github.community/t/android-projects-are-not-working/131143) that will require further investigation. That being said, we need to update this anyways, so I'm going to merge it and troubleshoot lgtm at a later time.

**Pull request merged**